docs/streams.md

```diff
@@ -5,91 +5,83 @@ EventStoreDB is purpose-built for event storage. Unlike traditional state-based
 These **events** are logically organized into **streams**, typically only one stream per entity. 
 
 ## Metadata and reserved names
+#### Stream metadata ####
 
-### Stream metadata
-
-Every stream in EventStoreDB has metadata stream associated with it, prefixed by `$$`, so the metadata stream
-from a stream called `foo` is `$$foo`. EventStoreDB allows you to change some values in the metadata, and you
-can write your own data into stream metadata that you can refer to in your code.
+In EventStoreDB, every stream and event is accompanied by metadata, distinguished by the **`$$`** prefix. For example, the stream metadata for a stream named **`foo`** is **`$$foo`**. You can modify specific metadata values and write your data to the metadata, which can be referenced in your code.
 
 ### Reserved names
 
-All internal data used by EventStoreDB is prefixed with a `$` character (for example `$maxCount` on a stream's
-metadata). Because of this you should not use names with a `$` prefix as event names, metadata keys, or stream
-names, except where detailed below.
+EventStoreDB includes a **`$`** prefix for all internal data, such as **`$maxCount`** in a stream's metadata. It's advisable to refrain from using names with a **`$`** prefix for event names, metadata keys, or stream names, except as detailed below.
```

Comment:
Can we say "Avoid using names..." as a compromise between the casual "should not" and "it's advisable"?